examples/mediadevices/src/index.js

```diff
@@ -1,19 +1,22 @@
 'use strict';
 
+// var Video = require('twilio-video');
 var Prism = require('prismjs');
 var getSnippet = require('../../util/getsnippet');
 var helpers = require('./helpers');
 var waveform = require('../../util/waveform');
-var applyAudioInputDeviceSelection = helpers.applyAudioInputDeviceSelection;
+var applyInputDeviceSelection = helpers.applyInputDeviceSelection;
 var applyAudioOutputDeviceSelection = helpers.applyAudioOutputDeviceSelection;
-var applyVideoInputDeviceSelection = helpers.applyVideoInputDeviceSelection;
 const connectWithSelectedDevices = helpers.connectWithSelectedDevices;
 const connectOrDisconnect = document.querySelector('input#connectordisconnect');
 const getRoomCredentials = require('../../util/getroomcredentials');
+const Video = require('twilio/lib/rest/Video');
```

Comment:
Why is this required?

Reply:
Oh, I believe that was added when I was trying to use Video.connect! WIll remove in the next commit. Thank you for catching that!